crates/bevy_pbr/src/pbr_material.rs

```diff
@@ -137,6 +137,180 @@ pub struct StandardMaterial {
     #[doc(alias = "specular_intensity")]
     pub reflectance: f32,
 
+    /// The amount of light transmitted _diffusely_ through the material (i.e. “translucency”)
+    ///
+    /// Implemented as a second, flipped [Lambertian diffuse](https://en.wikipedia.org/wiki/Lambertian_reflectance) lobe,
+    /// which provides an inexpensive but plausible approximation of translucency for thin dieletric objects (e.g. paper,
+    /// leaves, some fabrics) or thicker volumetric materials with short scattering distances (e.g. porcelain, wax).
+    ///
+    /// For specular transmission usecases with refraction (e.g. glass) use the [`StandardMaterial::transmission`] and
+    /// [`StandardMaterial::ior`] properties instead.
+    ///
+    /// - When set to `0.0` (the default) no diffuse light is transmitted;
+    /// - When set to `1.0` all diffuse light is transmitted through the material;
+    /// - Values higher than `0.5` will cause more diffuse light to be transmitted than reflected, resulting in a “darker”
+    ///   appearance on the side facing the light than the opposite side. (e.g. plant leaves)
+    ///
+    /// **Important:** The material's [`StandardMaterial::base_color`] also modulates the transmitted light.
+    ///
+    /// **Note:** Typically used in conjunction with [`StandardMaterial::thickness`]. To avoid self-shadows on complicated
+    /// mesh shapes without having to fine tune the thickness, consider using the [`NotTransmittedShadowReceiver`](crate::light::NotTransmittedShadowReceiver) component.
+    #[doc(alias = "translucency")]
+    pub diffuse_transmission: f32,
+
+    /// A map that modulates diffuse transmission via its **A channel**. Multiplied by [`StandardMaterial::diffuse_transmission`]
+    /// to obtain the final result.
+    ///
+    /// **Important:** The [`StandardMaterial::diffuse_transmission`] property must be set to a value higher than 0.0,
+    /// or this texture won't have any effect.
+    ///
+    /// ## Texture Packing
+    ///
+    /// Due to how channels are mapped, you can more efficiently use a single shared texture image
+    /// for configuring the following:
+    ///
+    /// - R - `transmission_texture`
+    /// - G - `thickness_texture`
+    /// - B - _unused_
+    /// - A - `diffuse_transmission_texture`
+    ///
+    /// The `KHR_materials_diffuse_transmission` glTF extension also defines a `diffuseTransmissionColorTexture`,
+    /// that _we don't currently support_. One might choose to pack the intensity and color textures together,
+    /// using RGB for the color and A for the intensity, in which case this packing advice doesn't really apply.
+    #[texture(17)]
+    #[sampler(18)]
+    pub diffuse_transmission_texture: Option<Handle<Image>>,
+
+    /// The amount of light transmitted _specularly_ through the material (i.e. via refraction)
+    ///
+    /// Implemented as a relatively expensive screen-space effect that allows ocluded objects to be seen through the material, while
```

Comment:
```
    /// Implemented as a relatively expensive screen-space effect that allows ocluded objects to be seen through the material, while
```

We should probably link to how to remove/alleviate the expensive performance costs if you don't need it. As it is, this is a warning that isn't actionable to the user. Excellent docs for the diffuse transmission though :)